sw/host/ot_certs/src/template/mod.rs

```diff
@@ -562,6 +571,11 @@ mod tests {
             authority_key_identifier: Value::variable("signing_pub_key_id"),
             subject_key_identifier: Value::variable("owner_pub_key_id"),
             basic_constraints: None,
+            key_usage: Some(KeyUsage {
```

Comment:
why is this addition necessary (as opposed to `key_usage: None,` )?

Also, if you want to maintain backwards compatibility so that the default key usage field value does not change after this patch shouldn't `cert_sign` be set here  instead of `key_agreement`?

Reply:
This just to make sure that the unittest is thorough (ie it can parse a key usage). I don't think this unit test should be use for the purpose of checking backward compatibility but maybe we could create another unittest whose sole purpose is to ensure that default values don't change?